.github/workflows/nightly.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: ['ubuntu-20.04', 'ubuntu-22.04', 'macos-11', 'macos-12']
+        os: ['ubuntu-20.04', 'ubuntu-22.04']  # 'macos-11', 'macos-12'
```

Comment:
Why is that?

Reply:
It is in the commit message: _CrateDB does not produce nightly tarballs for macOS._

Reply:
Sorry, I rushed and didn't see it. thx!